### ISSUE TITLE
Boost search results based on is_current / deprecated

### ISSUE THREAD

**Issue by thisisaaronland** [whosonfirst/whosonfirst-www-boundaryissues#40]
Boost current up; Boost deprecated down

**Comment by dphiffer** [comment]
Question: should `edtf:deprecated: "uuuu"` be treated the same as a situation where there's no `edtf:deprecated` property set at all? Presumably this relates to how search is ingesting the data.

**Comment by thisisaaronland** [comment]
Yes. `uuuu` (or possibly `XXXX` soon) means "unknown". Ultimately there shouldn't be an `edtf:deprecated` field absent an actual value since it is the data equivalent of a no/op.

**Comment by thisisaaronland** [comment]
Taking over to add/import splunker related tweaks.
